RickAndMorty/Views/RMEpisodeList/RMEpisodeHorizontalCell.swift

```diff
@@ -0,0 +1,163 @@
+        container.addSubview(airDateTitle)
+        airDateTitle.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            airDateTitle.topAnchor.constraint(equalTo: nameEpisodeLabel.bottomAnchor, constant: 10),
+            airDateTitle.leadingAnchor.constraint(equalTo: container.leadingAnchor, constant: 10)
+        ])
+        
+        airDateTitle.font = UIFont.preferredFont(forTextStyle: .headline)
+        airDateTitle.textColor = .black
+        airDateTitle.text = NSLocalizedString("Air Date: ", comment: "RMEpisodeHorizontalCell")
+        airDateTitle.setContentHuggingPriority(.required, for: .horizontal)
+    }
+    
+    private func airDateLabelView() {
+        container.addSubview(airDateEpisodeLabel)
+        airDateEpisodeLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            airDateEpisodeLabel.topAnchor.constraint(equalTo: nameEpisodeLabel.bottomAnchor, constant: 10),
+            airDateEpisodeLabel.leadingAnchor.constraint(equalTo: airDateTitle.trailingAnchor, constant: 10)
+        
+        ])
+        
+        airDateEpisodeLabel.font = UIFont.preferredFont(forTextStyle: .body)
+        airDateEpisodeLabel.textColor = .black
+        airDateEpisodeLabel.numberOfLines = 0
+    }
+    
+    private func createdView() {
+        container.addSubview(createdTitle)
+        createdTitle.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            createdTitle.topAnchor.constraint(equalTo: airDateTitle.bottomAnchor, constant: 10),
+            createdTitle.leadingAnchor.constraint(equalTo: container.leadingAnchor, constant: 10),
+            createdTitle.bottomAnchor.constraint(equalTo: container.bottomAnchor, constant: -20)
+        ])
+        createdTitle.font = UIFont.preferredFont(forTextStyle: .headline)
+        createdTitle.textColor = .black
+        createdTitle.setContentHuggingPriority(.required, for: .horizontal)
+        createdTitle.text = NSLocalizedString("Created: ", comment: "RMEpisodeHorizontalCell")
+    }
+    
+    private func createdLabelView() {
+        container.addSubview(createdEpisodeLabel)
+        createdEpisodeLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            createdEpisodeLabel.topAnchor.constraint(equalTo: airDateEpisodeLabel.bottomAnchor, constant: 10),
+            createdEpisodeLabel.leadingAnchor.constraint(equalTo: createdTitle.trailingAnchor,constant: 10),
+            createdEpisodeLabel.bottomAnchor.constraint(equalTo: container.bottomAnchor, constant: -20)
+        ])
+        createdEpisodeLabel.font = UIFont.preferredFont(forTextStyle: .body)
+        createdEpisodeLabel.textColor = .black
+        createdEpisodeLabel.setContentHuggingPriority(.required, for: .horizontal)
+    }
+    
+}
+
+extension RMEpisodeHorizontalCell {
+    
+    override func prepareForReuse() {
```

Comment:
move this function right down to `setup` function

Reply:
Done